.github/workflows/js-ci.yml

```
@@ -265,17 +261,21 @@ jobs:
         run: pnpm --fail-if-no-match --filter ${{ env.WORKSPACE }} cy:ldap-server &
         working-directory: js
 
-      - name: Run Cypress
-        uses: cypress-io/github-action@v6
+      - name: Install Playwright browsers
+        run: pnpm --fail-if-no-match --filter ${{ env.WORKSPACE }} exec playwright install --with-deps
+        working-directory: js
+
+      - name: Run Playwright tests
+        run: pnpm --fail-if-no-match --filter ${{ env.WORKSPACE }} test
```

This is currently running the unit tests for the Admin Console, which have already been executed in the `admin-ui` job. It should be running the integration tests instead.
```suggestion
        run: pnpm --fail-if-no-match --filter ${{ env.WORKSPACE }} test:integration
```